ProcessManager.py

```diff
@@ -66,47 +77,40 @@ def createProcess(self, name, command):
 
         self.processes[name] = RoboProcess(command)
 
-    # TODO: Do we really need this if we have createProcess?
-    def createThread(self):
+
+    def start(self,**name):
         """
-        TODO create a function that takes in parameters and uses those parameters to
-        ceate one single thread. The thread must then be stored in "self.T".
+        Starts processes
+
+        Arguments:
+        name - name or set of names to identify process, must be unique to process manager.
         """
-        pass
-
-    def startProcess(self, name):
-        """Start a single process."""
-        print('Starting {}'.format(name))
-        self.processes[name].execute()
-
-    def startAllProcesses(self):
-        """Start all managed processes."""
-        for process in self.processes:
-            self.startProcess(process)
-
-    def stopProcess(self, name):
-        """Stop a process by name."""
-        print ("Shutting down " + name)
-        self.processes[name].popen.kill()
-        self.processes[name].popen.wait()
-
-    def stopAllProcesses(self):
-        """Stop all managed processes."""
-        print ("Shutting down")
-        for process in self.processes:
-            self.stopProcess(process)
-
-    def status(self, thread):
-        """Return the status of a thread or process."""
-        raise NotImplementedError()
 
-    def verify(self, thread):
-        """Verify if a thread of process is running properly."""
-        raise NotImplementedError()
+        if name == []:
+            """Start all managed processes."""
```

Comment:
Comments that are not docstrings should use the `#` symbol instead of tripple quoted strings. https://www.python.org/dev/peps/pep-0008/#block-comments